### PR TITLE
New carriers pages docs

### PR DESCRIPTION
**Pull request by boherm**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 9.x
| Description?  | This PR aim to add some tech docs about new Carriers pages in BackOffice.
| Fixed ticket? | Fixes https://github.com/PrestaShop/PrestaShop/issues/10586

### CODE REVIEW

**Review by tleon — APPROVED** [review]

File: development/page-reference/back-office/carrier/add-new-carrier/_index.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+### Shipping locations & costs
+
+This tab allows you to set the carrier zones, countries, prices, free shipping, etc.
+
+{{< figure src="./img/shipping-locations-and-costs-tab.png" title="Shipping locations and costs tab" >}}
+
+The definition of this form can be found in [src/PrestaShopBundle/Form/Admin/Improve/Shipping/Carrier/ShippingLocationsAndCostsType.php](https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Form/Admin/Improve/Shipping/Carrier/ShippingLocationsAndCostsType.php).
+
+All the behavior of this tab is manage by `carrier-form-manager.ts` file, that is located in [admin-dev/themes/new-theme/js/pages/carrier/form/carrier-form-manager.ts](https://github.com/PrestaShop/PrestaShop/blob/develop/admin-dev/themes/new-theme/js/pages/carrier/form/carrier-form-manager.ts).
```
Comment (matks) [inline development/page-reference/back-office/carrier/add-new-carrier/_index.md]:
```suggestion
All the behavior of this tab is managed by `carrier-form-manager.ts` file, that is located in [admin-dev/themes/new-theme/js/pages/carrier/form/carrier-form-manager.ts](https://github.com/PrestaShop/PrestaShop/blob/develop/admin-dev/themes/new-theme/js/pages/carrier/form/carrier-form-manager.ts).
```

File: development/page-reference/back-office/carrier/add-new-carrier/_index.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+---
+title: Add new carrier
+weight: 2
+menuTitle: Add new carrier
+---
+
+# Add new carrier
+
+Page can be reached by visiting `Shipping -> Carriers -> Add new carrier` in the admin panel. It allows the Back Office user to create a new carrier.
```
Comment (matks) [inline development/page-reference/back-office/carrier/add-new-carrier/_index.md]: The page can also be used to edit an existing carrier 😉

File: development/page-reference/back-office/carrier/add-new-carrier/_index.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+{{% notice %}}
+
+Carrier can change its ID when it is edited. In fact, if you edit a carrier that is already associated with an order, the carrier ID will change automatically.
```
Comment (matks) [inline development/page-reference/back-office/carrier/add-new-carrier/_index.md]:
```suggestion
When you edit a carrier, sometimes the data of the carrier is simply altered, sometimes a copy of the data is done, the previous data is de-activated but preserved, and your modifications are done on the new copy. This happens when a customer has purchased an order using this carrier, in order to preserve the carrier data as it was during the purchase.
```

**Review by matks — COMMENTED** [review]
Thanks ;) please find 1 typo and 2 improvements

**Review by matks — APPROVED** [review]
Good for me

I see it's still a draft. i think we can nevertheless merge it and update it later, wdyt?

**Comment by boherm** [comment]
Done! Thanks guys ;)

**Pull request merged**